src/elements/content-sidebar/AddTaskMenu.scss

```diff
@@ -0,0 +1,29 @@
+@import '../common/variables';
+
+$bsc-AddTaskMenu-width: 256px;
+
+.bsc-AddTaskMenu {
+    padding: 0;
+}
+
+.bsc-AddTaskMenu-menuItem {
+    display: flex;
+    max-width: $bsc-AddTaskMenu-width;
+    min-height: 80px;
+    padding: 20px;
+    white-space: normal;
+}
+
+.bsc-AddTaskMenu-avatar {
+    margin-right: 4px;
+}
+
+.bsc-AddTaskMenu-title {
+    color: $threes;
```

Comment:
This might be better as `bdl-neutral-01`. can you verify use of #333333 with design?

Reply:
Yes, #333 is used in design

Reply:
we may be moving these to the neutral 01 color at some point, so I think it's ok to keep this one as-is since that is ongoing